build.sh

```diff
@@ -121,8 +121,10 @@ echo "Extracting Python framework..."
 
 mkdir -p "$PACKAGE/Frameworks/Python.framework"
 
+7z x -so "$DL/$PYTHON_PKG" Python_Framework.pkg/Payload | wc
+7z x -so "$DL/$PYTHON_PKG" Python_Framework.pkg/Payload | zcat | wc
```

Comment:
are these lines leftover from debugging?

Reply:
yep. will update with latest commit

Reply:
@jannau updated to remove debug code